spec/un_locode_spec.rb

```diff
@@ -94,10 +94,17 @@
       let!(:country) { UnLocode::Country.create name: 'NETHERLANDS', code: 'NL' }
       let!(:location) { UnLocode::Locode.create name: 'Venlo', city_code: 'VEN', country: country }
 
-      context 'with supported functions' do
+      context 'with supported functions and space in search' do
         subject { UnLocode::Locode.find_by_locode(search_term) }
         let(:search_term) { 'NL VEN' }
-
+        its(:city_code) {should eql('VEN') }
+        its(:name) { should eql('Venlo') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.